content/influxdb/cloud-iox/union.md

```diff
@@ -0,0 +1,74 @@
+---
+title: The UNION clause
+list_title: The UNION clause
+description: > 
+    Use the `UNION` clause to combine the results of two or more queries into a single set of results.
+menu:
+  influxdb_cloud_iox:
+    name: The UNION clause
+    parent: Explore data using SQL
+weight: 295
+---
+
+The `UNION` clause combines the results of two or more `SELECT` statements into a single results set.
+
+Key points to keep in mind when using the `UNION` clause:
+
+ - the number of columns in the column list must be the same
+ - the columns in the column list must be in the same order
+ - the data types in the column list must be the same or compatible  
```

Comment:
```suggestion
**Note when using the `UNION` clause**:

- The number of columns in each result set must be the same
- Columns must be in the same order
- Columns must be the same or compatible data types
```

Where are these guidelines coming from? Just from testing? In my own testing, columns are unioned/merged by their position in the first select statement. The value of the first column in the second result set is used to populate the first column of the unioned result set, but the name of the column is always from the first set.